src/DynamicNLPModels.jl

```diff
@@ -631,16 +638,30 @@ function _build_dense_lq_dynamic_model(dnlp::LQDynamicData{T,V,M,MK}) where {T,
     LinearAlgebra.axpy!(-1, As0_bounds, ucon2)
     LinearAlgebra.axpy!(-1, As0_bounds, lcon2)
 
-    lcon = vcat(lcon, vec(lcon2))
-    ucon = vcat(ucon, vec(ucon2))
+    lcon = (similar(Q, length(lcon1) + length(lcon2)) .= 0)
```

Comment:
I would prefer the syntax:
```julia
lcon = similar(Q, n) ; fill!(lcon, 0)
```

Reply:
@frapac, if I am just initiating an array to fill it through `mul!`, I do not need to do `fill!()`, correct?